src/components/grid/Column.js

```diff
@@ -12,82 +8,81 @@ const defaultShape = {
 
 const Column = React.createClass({
   propTypes: {
-    breakpoints: React.PropTypes.shape(defaultShape),
-    columnCount: React.PropTypes.number,
     offset: React.PropTypes.shape(defaultShape),
-    order: React.PropTypes.shape(defaultShape),
```

Comment:
I didn't see an immediate way to implement it and I didn't think we were using it anywhere yet so I pruned it for the time being to get things moving.